controllers/UsersController.js

```diff
@@ -83,4 +85,63 @@ export default class UsersController {
       }
     }).catch(next);
   }
+
+  /** @param {Object} req - Request object
+  * @param {Object} res - Response object
+  * @returns {Object} success or failure response
+  */
+  static sendEmail(req, res) {
+    let transport;
+    const mailOptions = {
+      from: 'sinmi.mtracker@gmail.com',
+      to: req.body.user.email,
+      subject: 'Password reset Authors-Haven.com',
+      html: `<h3 style="color: blue;">Author's Haven</h3><p>Hello ${req.body.user.username},</p><p> You can reset your password by clicking this <a href='${req.protocol}://${req.headers.host}/api/users/password/reset/edit?token=${req.body.token}'>link</a>. You received this email because you requested for password reset</p>
+      <div style="text-align: center; background: aliceblue;"> <h5 style="padding: 10px;color: blue">Author's Haven &copy;2018</h5></div>`
+    };
+    if (process.env.NODE_ENV === 'TEST' || process.env.NODE_ENV === 'test') {
```

Comment:
I don't think this should be in the userController. Instead, let's have it in a helper method.